aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1056,11 +1057,7 @@ module aptos_token::token {
             largest_property_version: 0,
             supply: 0,
             uri,
-            royalty: Royalty {
-                royalty_points_denominator,
-                royalty_points_numerator,
-                payee_address: royalty_payee_address,
-            },
+            royalty: create_royalty(royalty_points_numerator, royalty_points_denominator, royalty_payee_address),
```

Comment:
this then becomes a breaking change

Reply:
After deleting the assertion, this is the same as mainnet.  no enforcement on royalty denominator to be non-zero.

The create_royalty is  only on main branch which has not been released to mainnet yet.  So this should not be a breaking change  now.